### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by annastuchlik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,147 +85,96 @@ If you are also interested in uploading other outputs and chain them with your w
...
+Alternatively, you can provide workspace-relative path to the .properties file where your test configuration is defined:
 
 ```yaml
     - name: Run dotTEST analyzer
-      # You may pin to the exact commit or the version.
-      # uses: tobyash86/run-dottest-analyzer-proto@1bc4be095189f455793afdb10b47127e06ae25ff
       uses: tobyash86/run-dottest-analyzer-proto@prda0.1.6a
       with:
-        # An URL of test configuration to be used for analysis
         config: ".\.dottest\MyTestConfig.properties"
```
Comment (tobyash86) [inline README.md:127]: please improve display of this text (fix red highlight)

File: README.md [hunk 2]
```diff
@@ -92,147 +85,96 @@ If you are also interested in uploading other outputs and chain them with your w
...
+You configure the `solution` parameter to provide the path to the solution you want to analyze. This parameter is particularly useful when your workflow has a complex structure and includes several solution files. You can specify one or more solutions by providing semicolon-separated paths to solution files.
 
 ```yaml
     - name: Run dotTEST analyzer
-      # You may pin to the exact commit or the version.
-      # uses: tobyash86/run-dottest-analyzer-proto@1bc4be095189f455793afdb10b47127e06ae25ff
       uses: tobyash86/run-dottest-analyzer-proto@prda0.1.6a
       with:
-        # Path to solution(s) to be analyzed. Specify multiple times to analyze many solutions. Supports ANT-style wildcards.
         solution: ".\src\*.sln"
```
Comment (tobyash86) [inline README.md:137]: please improve display of this text (fix red highlight)

**Review by tobyash86 — CHANGES_REQUESTED** [review]
Please review my comments

File: README.md [hunk 3]
```diff
@@ -92,147 +85,96 @@ If you are also interested in uploading other outputs and chain them with your w
...
         config: ".\.dottest\MyTestConfig.properties"
 ```
 
-assuming that provided path is relative to workspace root.
-
-### How do I fine tune the scope of the analysis
-
-This option is useful when workflow contains many solution files, has complex structure or you are interested in analyzing only part of it:
+#### Defining the Scope for Analysis
+You configure the `solution` parameter to provide the path to the solution you want to analyze. This parameter is particularly useful when your workflow has a complex structure and includes several solution files. You can specify one or more solutions by providing semicolon-separated paths to solution files.
```
Comment (tobyash86) [inline README.md:131]: I would remove last sentence as next section is entirely about that

File: README.md [hunk 4]
```diff
@@ -80,9 +74,8 @@ To upload it, add to your workflow the following sarif upload action (which is p
         sarif_file: ${{ steps.dottest.outputs.report }}
 ```
 
-#### I wish to upload my job artifacts and bind them with my workflow run
-
-If you are also interested in uploading other outputs and chain them with your workflow run, you can use another [action](https://github.com/actions/upload-artifact) to do that:
+#### Adding Other Job Artifacts the a GitHub Workflow
```
[the remark this answers is not in the document]
Comment (annastuchlik) [reply README.md]: fixed

File: README.md (hunk 1 above)
Comment (annastuchlik) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (annastuchlik) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (annastuchlik) [reply README.md]: removed

**Review by tobyash86 — APPROVED** [review]

**Pull request merged**